src/utils/version.ts

```
@@ -0,0 +1,10 @@
+/**
+ * Application version injected at build time via Vite's define.
+ * Format: YYYY.MM.DD-shortSha (e.g., 2025.01.15-abc1234)
+ * Falls back to 'dev' when running locally without version injection.
+ */
+export const APP_VERSION: string =
+  typeof __APP_VERSION__ !== 'undefined' ? __APP_VERSION__ : 'dev';
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)